### PR TITLE
feat: input vars and prevent injections

### PR DESCRIPTION
**Pull request by Glenn-Terjesen**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -52,15 +63,18 @@ jobs:
       image_name: ${{ env.GHA_DOCKER_PUSH_IMAGE_NAME }}
```
Comment (AlexanderBrevig) [inline .github/workflows/push.yml:63]: This is still vulnerable. 

Change from `image_name: ${{ env.GHA_DOCKER_PUSH_IMAGE_NAME }}` to `image_name: ${GHA_DOCKER_PUSH_IMAGE_NAME}`

**Review by AlexanderBrevig — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -41,42 +41,40 @@ jobs:
...
-          name: ${{ env.GHA_DOCKER_BUILD_ARTIFACT_NAME }}
-          path: ${{ env.GHA_DOCKER_BUILD_ARTIFACT_PATH }}
+          name: "${GHA_DOCKER_BUILD_ARTIFACT_NAME}"
+          path: "${GHA_DOCKER_BUILD_ARTIFACT_PATH}"
       - uses: docker/setup-buildx-action@v3.3.0
       - id: build
         uses: docker/build-push-action@v5.3.0
         with:
-          file: ${{ env.GHA_DOCKER_BUILD_DOCKERFILE }}
+          file: "${GHA_DOCKER_BUILD_DOCKERFILE}"
```
Comment (Glenn-Terjesen) [inline .github/workflows/build.yml:68]: fungerer det her med with: ? tror ikke det

Comment (AlexanderBrevig) [reply .github/workflows/build.yml:68]: det... godt spørsmål

Comment (AlexanderBrevig) [reply .github/workflows/build.yml:68]: ser ikke sånn ut nei...
haha
I fix

**Review by AlexanderBrevig — APPROVED** [review]

**Pull request merged**